### PR TITLE
add spanish translation for scope

### PR DESCRIPTION
**Pull request by pepoviola**
Hi @sethvincent, this is the Spanish translation for scope challenge.

Thanks!

### CODE REVIEW

**Comment by SomeoneWeird** [comment]
.. this doesn't look spanish to me?

File: problems/scope/problem_es.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+	bar(b * 4);
+}
+
+foo(); // 4, 2, 48
+```
+IIFE, Immediately Invoked Function Expression( Expresión de Functión Invocada Inmediatamente ), es un patrón común para crear ámbitos locales.
+Por ejemplo:
+```js
+	(function(){ // La expresión de la función está entre parentesis
```
Comment (a0viedo) [inline problems/scope/problem_es.md]: parentesis -> paréntesis

**Comment by a0viedo** [comment]
aside the emphasis, looks good to me. awesome work @pepoviola 

would be nice if someone else could review this too /cc @eafelix @mgtitimoli @pricco @julianduque

File: problems/scope/problem_es.md (hunk 1 above)
Comment (pepoviola) [inline problems/scope/problem_es.md]: nice catch! fixed now.
Thx!

**Comment by theHazzard** [comment]
looks good to me!

**Comment by gaboesquivel** [comment]
@a0viedo  gracias por trabajar en esto, nos servirá para el nodeschool international day :+1:

**Comment by a0viedo** [comment]
merged, thanks @theHazzard 

@gaboesquivel esa es la idea :) queremos que se haga el release de javascripting con las traducciones a npm

**Pull request merged**